core/store/src/trie/mod.rs

```diff
@@ -376,6 +376,15 @@ impl RawTrieNode {
             _ => Err(std::io::Error::new(std::io::ErrorKind::Other, "Wrong type")),
         }
     }
+
+    #[cfg(test)]
+    pub(crate) fn get_key(&self) -> Option<&[u8]> {
+        match self {
+            RawTrieNode::Leaf(key, _, _) => Some(key),
+            RawTrieNode::Extension(key, _) => Some(key),
+            _ => None,
```

Comment:
```suggestion
            RawTrieNode::Branch(_, _) => None,
```

This way if we add more variants to RawTrieNode, the compiler will guide us in finding all the places we need to consider making adjustments to.

Reply:
really good point!